docs/pywbemclicommands.rst

```diff
@@ -979,14 +979,7 @@ options for pywbemcli. The data includes:
   received from the WBEM server during connection setup. See :ref:`--no-verify general option`.
 * **certfile** - optional server certificate filename. See :ref:`--certfile general option`.
 * **keyfile** - optional client private keyfile filename. See :ref:`--keyfile general option`.
-* **use-pull** - optional parameter that defines whether pull operations are
-  to be required, used if they exist or not used. See :ref:`--use-pull general option`.
-* **pull-max-cnt** - optional count of object per pull operation.  See :ref:`--pull-max-cnt general option`.
```

Comment:
I think use-pull and pull-max-cnt should be part of the data stored for a connection definition, because they represent how to interact with the WBEM server referenced by the connection definition.

Reply:
We agreed that they wern't the other day.  Since the default is to try pull and then go to the traditional, the only real reason for this option is special cases of testing by forcing a pull, etc. 
The reasons I see for doing this is for the user to test response time with pull vs no pull, etc.
 Even a server that does not sujpport pull will work OK with just the overhead of the pull tested first. 

Of course, on the other hand, if it becomes part of the connection itself, it means that the extra pull request can be completely turned off for a particular server.

In reality, this one should probably exist in both camps.  Apply to a particular server but override with in general but I am not sure that I can really do that.  

At the same time, that then brings up the question of the pull_max_cnt. It is part of the server or part of the general ops.

**DISCUSSION**

Reply:
I am ok with not making it part of the connection definition. CLOSED.